controller/handler_ctrl/bind.go

```diff
@@ -56,9 +64,21 @@ func (self *bindHandler) BindChannel(binding channel.Binding) error {
 	binding.AddPeekHandler(trace.NewChannelPeekHandler(self.network.GetAppId(), binding.GetChannel(), self.network.GetTraceController(), traceDispatchWrapper))
 	binding.AddPeekHandler(metrics2.NewCtrlChannelPeekHandler(self.router.Id, self.network.GetMetricsRegistry()))
 
```

Comment:
This should be the actual 0.25 version this will release with

Reply:
I gotcha, I thought this was for when the heartbeat handler was added to make sure it worked. Using `0.25.5`